### ISSUE TITLE
MacOS Catalina 10.15.4. zsh: Bad CPU type in executable

### ISSUE THREAD

**Issue by MarkHunnibell** [ImageMagick/ImageMagick#1911]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am using the latest version of ImageMagick
- [x] I have searched [open](https://github.com/ImageMagick/ImageMagick/issues) and [closed](https://github.com/ImageMagick/ImageMagick/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

I do not use ImageMagick frequently, but have had not problems until apparently after a recent update to MacOS Catalina 10.15.4. The default shell for Catalina 10.15.4. is now zsh and I am using that.

However, any time I try to run any ImageMagick commands from the Terminal window, I get this error:

`Bad CPU type in executable`

For example, when attempting a simple convert command, it fails with:

`zsh: bad CPU type in executable: convert`

If I use bash shell instead (just by typing "bash" <Enter>), I get a similar error:

`bash: /usr/local/bin/convert: Bad CPU type in executable`

It gives me the same error when I try to verify my ImageMagick installation:

`bash: /usr/local/bin/identify: Bad CPU type in executable`

When I encountered this first error, since it had been a while since I installed/updated ImageMagick, I used homebrew to re-install it, which it seemed to do without much fuss (I could not keep track of all the various notes on prerequisites as it installed, but it did not seem to fail anywhere). So I am assuming I am using the most current version.

I think this is likely a simple issue to resolve. I searched for comments here and in Google and could not find any relevant solutions.

If you do have a solution, please tell me step-by-step so that nothing is lost in translation from Expert to Novice.

### Steps to Reproduce

See above.

### System Configuration

MacOS Catalina 10.15.4. Terminal

- ImageMagick version: (unknown, but presumably latest production. see above)
- Environment (Operating system, version and so on): MacOS Catalina 10.15.4. 
- Additional information:

**Comment by urban-warrior** [comment]
Try the `magick` command, `/usr/local/bin/magick`.  Does that work for you?  Catalina switched to zsh and the exception you receive may be related to zsh vs. bash.

**Comment by MarkHunnibell** [comment]
Nope. It does not exist:

**% magick**
`zsh: command not found: magick`

**% /usr/local/bin/magick**
`zsh: no such file or directory: /usr/local/bin/magick`

I just re-ran the `brew install ImageMagick` command again and it ran but with fewer updates to prerequisites. The shorter output allowed me to see there was information about bad symbolic links and suggested...

```
To force the link and overwrite all conflicting files:
  brew link --overwrite imagemagick
```
So I did that and the answer was:

`Linking /usr/local/Cellar/imagemagick/7.0.10-7... 70 symlinks created`

Then I tried ImageMagick commands with and without the full path. 

**They worked.**

So... problem solved for me, but for others... maybe the script could be smartened up to PAUSE at that point in the install and just ASK if it should overwrite existing symbolic links.
